src/test/resources/expected-classes/jackson-resolver-union/GraphqlJacksonTypeIdResolver.java.txt

```diff
@@ -0,0 +1,49 @@
+package com.kobylynskyi.graphql.unionresolver;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.DatabindContext;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.jsontype.impl.TypeIdResolverBase;
+
+@javax.annotation.Generated(
+    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
+    date = "2020-12-31T23:59:59-0500"
+)
+public class GraphqlJacksonTypeIdResolver extends TypeIdResolverBase {
+
+    private JavaType superType;
+
+    @Override
+    public void init(JavaType baseType) {
+        superType = baseType;
+    }
+
+    @Override
+    public JavaType typeFromId(DatabindContext context, String typename) {
+        try {
+            var clazz = Class.forName(
```

Comment:
can it use `var` here  in java 8?

Reply:
No, in Java 8 it is not possible

Reply:
I have update in my pr.

Reply:
Yes, good catch.
My env was using Java11 at first, then I switched back to Java 8.

Reply:
Good catch indeed. Thanks for fixing this.